docs/source/quantization.rst

```diff
@@ -14,12 +14,12 @@ Introduction to Quantization
 
 Quantization refers to techniques for performing computations and storing
 tensors at lower bitwidths than floating point precision. A quantized model
-executes some or all of the operations on tensors with integers rather than
+executes some or all of the operations on tensors with reduced precision rather than
 floating point values. This allows for a more compact model representation and
```

Comment:
"rather than floating point values" should this be changed, since you changed the wording from integers->reduced precision. I thought we have some fp16 quantization stuff?

Reply:
makes sense, I think floating point should be full precision in this case

Reply:
yeah fp16 is also considered reduced precision